plugins/home/src/homePageComponents/QuickStart/Content.tsx

```diff
@@ -79,6 +81,7 @@ export const Content = (props: QuickStartCardProps): JSX.Element => {
         </Grid>
       </Grid>
       {props.video && props.video}
+      {props.additionalContent && props.additionalContent}
```

Comment:
Should it be something like this?

```suggestion
      {(props.additionalContent && props.additionalContent) || (props.video && props.video)}
```

And then you would removed line 83. If we keep it the way it is now you could provide values for `video` and `additionalContent` which isn't the desired long term goal where we would remove `video`.